### DISCUSSION TITLE
In app chat architect discussion

### DISCUSSION THREAD

**Discussion by Pjaijai** [Pjaijai/Referalah#150] [討論 | General]
Chat box => supabase
Notification => firebase (free)

Open to any solution or tech stack
Will update design idea and flow later

**Comment by marukosy124** [comment]
Just want to confirm, are you looking for real-time data transfer or API calls to send and receive messages?
e.g. in-app chat in Indeed is simply API calls (from what I observed)

Also is there anything else, other than text, that users can send and receive? 
e.g. attach files
